production.qmd

```diff
@@ -0,0 +1,56 @@
+---
+title: "Production"
+---
+
+## Beyond Community
+
+Available at <https://production.r-multiverse.org>, the Production repository is designed to go one step beyond Community.
+
+It is comprised only of Community packages that pass additional checks, including `R CMD check` on all major platforms, ensuring a cohort of packages that work seamlessly togther.
+
+So whilst Community will always offer the latest packages, as released by maintainers, they will only make it to Production if they pass these additional quality controls.
+
+## A Snapshot Model for Production
+
+But we take it one step further.
+
+Instead of the continuous rolling deployment model that CRAN provides, we provide snapshots of fixed package versions. This has the advantage that:
+
+1. All packages within a snapshot are guaranteed to work with each other.
+ + At any one point in time on CRAN, there will be packages that fail checks but are within grace periods and have not yet been updated or archived.
+2. Using fixed package versions aids reproducibility.
+3. Suited to environments without the ability to update over the internet.
+
+We take quarterly snapshots on the 15th of each month in May, August, November and February.
```

Comment:
genuine question. should people count on the snapshots to be maintained forever or should they download them.

Reply:
I think this is still an open question, whether we will maintain an archive. So there is no answer yet (at least for the website).